tsconfig.json

```diff
@@ -26,11 +26,8 @@
       "#internal/nitro": [
         "./node_modules/nitropack/dist/runtime"
       ],
-      "#internal/nitro/app": [
-        "./node_modules/nitropack/dist/runtime/app"
-      ],
-      "#internal/nitro/utils": [
-        "./node_modules/nitropack/dist/runtime/utils"
+      "#internal/nitro/*": [
+        "./node_modules/nitropack/dist/runtime/*"
```

Comment:
Same for here about assumptions of the internal runtime directory structure. It should be avoided when possible.